.github/workflows/quant_scholar.yml

```
@@ -0,0 +1,33 @@
+name: run quant_scholar
+
+on:
+  schedule:
+    - cron: '0 0 */2 * *'
+  workflow_dispatch:
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo-b
+        uses: actions/checkout@v3
+        with:
+          repository: LLMQuant/quant-wiki
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Fetch README.md from quant-scholar
+        run: |
+          curl -o README.md https://raw.githubusercontent.com/LLMQuant/quant-scholar/main/README.md
+
+      - name: Replace docs/scholar/index.md with README.md
+        run: |
+          mv README.md docs/scholar/index.md
+
+      - name: Commit and push changes
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"
+          git add docs/scholar/index.md
+          git commit -m "Automated update of paper list from quant-scholar"
```

Perhaps we could use a more informative commit message to indicate that this is a workflow.
```suggestion
          git commit -m "[Workflow] Automated update of paper list from quant-scholar"
```

---

Solved.